helm/prometheus/values.yaml

```diff
@@ -121,6 +121,9 @@ rules:
 secrets: {}
 
 service:
+  ## Maintains session affinity.  Should be set to ClientIP for HA setup
+  ## Only options are ClientIP and None.  Do not leave blank.
+  sessionAffinity: ClientIP
```

Comment:
I'm afraid to change the default behaviour for ClientIP because we already have other users using None.  WDYT to rollback this value to None?